components/InputEmail.js

```diff
@@ -0,0 +1,19 @@
+export default function InputEmail({ inputDisabled, handleChange }) {
+  return (
+    <div>
+      <h4>3. Enter Email Address</h4>
+      <div className="cdx-text-input input-group">
+        <input
+          className="cdx-text-input__input"
+          id="email"
+          name="email"
+          type="email"
+          disabled={inputDisabled}
+          placeholder="user@gmail.com"
```

Comment:
Please change this to: "Enter e-mail address (optional)"

Reply:
@coderwassananmol 
fixed